listenbrainz/webserver/templates/navbar.html

```diff
@@ -72,21 +73,33 @@
               {{ current_user.musicbrainz_id }} <span class="caret"></span>
             </a>
             <ul class="dropdown-menu" role="menu">
-              <li><a href="{{ url_for('user.profile', user_name=current_user.musicbrainz_id) }}">My Listens</a></li>
-              <li><a href="{{ url_for('index.feed') }}">My Feed</a></li>
+              <li><a href="{{ url_for('profile.info') }}">My profile</a></li>
               <li><a href="{{ url_for('user.playlists', user_name=current_user.musicbrainz_id) }}">My Playlists</a></li>
-              <li><a href="{{ url_for('recommendations_cf_recording.info', user_name=current_user.musicbrainz_id) }}">Tracks you might like</a></li>
               <li><a href="{{ url_for('profile.music_services_details') }}">Connect Services</a></li>
               <li><a href="{{ url_for('profile.import_data') }}">Import listens</a></li>
               <li><a href="{{ url_for('profile.export_data') }}">Export</a></li>
-              <li><a href="{{ url_for('profile.info') }}">My profile</a></li>
               <li><a href="{{ url_for('login.logout') }}">Sign out</a></li>
             </ul>
           </li>
         {%- endif -%}
       </ul>
     {% endif %}
+      <form id="searchForm" class="navbar-form">
+          <input id="searchQuery" type="text" class="form-control input-sm" placeholder="Go to Profile">
+        <button type="submit" class="btn input-sm"><span class="glyphicon glyphicon-search"></span></button>
+      </form>
     </div>
-
   </div>
 </nav>
+<script type="text/javascript">
+  const query = document.getElementById('searchQuery');
```

Comment:
No, need to do it this way. Change the form action to /search and method to get. For the empty check, make the query input required?

Reply:
@amCap1712 I was unable to make a url modification through that because that makes a query and appends that to the url instead of modifying it.

Reply:
Oh sorry. I should have added more context. I have implemented actual user search in a different PR. Let me finish that up and see how we can fix this up.

Reply:
Search box has now been added to #1628